src/DIRAC/DataManagementSystem/Service/TornadoFileCatalogHandler.py

```diff
@@ -22,6 +22,7 @@
 import os
 
 from io import BytesIO
+from six import StringIO
 # from DIRAC
 from DIRAC.Core.DISET.RequestHandler import getServiceOption
```

Comment:
This, and other imports in this module don't seem to be used.

Reply:
I've cleaned them